extensions/oplogPopulator/OplogPopulatorTask.js

```diff
@@ -19,8 +19,17 @@ werelogs.configure({ level: config.log.logLevel,
 
 const mongoConfig = config.queuePopulator.mongo;
 const oplogPopulatorConfig = config.extensions.oplogPopulator;
-// Notification extension is the only one using OplogPopulator for now
-const activeExtensions = config.extensions.notification ? ['notification'] : [];
+const activeExtensions = [];
+
+[
+    'notification',
+    'lifecycle',
+    'replication',
+].keys(config.extensions).forEach(ext => {
+    if (config.exntensions[ext]) {
+        activeExtensions.push(ext);
+    }
+});
```

Comment:
does this really work? You are trying to push into a `const` variable, this seems fishy...

It seems however to be a simple use-case for `filter` :
```suggestion
const activeExtensions = [
    'notification',
    'lifecycle',
    'replication',
].filter(ext => config.extensions[ext]);
```

Reply:
> does this really work? You are trying to push into a const variable, this seems fishy...

yes. javascript quirk, the `const` is to the reference, the underlying array can still be populated
using your filter suggestions